.gitignore

```diff
@@ -48,6 +48,7 @@ Icon?
 # documentation files
 docs/source/generated
 docs/source/generated-notebooks
+*.org
```

Comment:
I see what you did there.

Reply:
I suppose I could've just added an override to have `org-mode` treat the `.egg` suffix like a org file?

Reply:
🤷🏻 I personally do not mind this here.